source/basic.tex

```diff
@@ -3696,10 +3696,17 @@
 
 \pnum
 \indextext{storage duration!static}%
-All variables which do not have dynamic storage duration, do not have thread
-storage duration, and are not local
-have \defn{static storage duration}. The
-storage for these entities lasts for the duration of the
+All variables which
+\begin{itemize}
+\item
+do not have thread storage duration and
+\item
+belong to a namespace scope\iref{basic.scope.namespace} or
+are declared with
```

Comment:
Would it be useful to say "are first declared" here, given that `static` is inherited in redeclarations (and can't be present in a redeclaration of a class member)?

Reply:
Yes, that clarifies the "class member redeclaration" case.

Fixed